### ISSUE TITLE
Update to prettier 1.7.0.

### ISSUE THREAD

**Issue by gb714us** [prettier/prettier-vscode#221]
There is a new version of prettier out now.

https://github.com/prettier/prettier/releases/tag/1.7.0

**Comment by azz** [comment]
You are able to install your own version of prettier locally and this extension will use it. Is there a specific feature you would like added to the extension that is enabled by the 1.7 release?

**Comment by gb714us** [comment]
None in particular. I just noticed that it the plugin always uses the newest version, just something I thought I'd let the maintainers be aware about. =)

**Comment by RobinMalfait** [comment]
If you provide a PR I'll accept it immediately 👍  😉

**Comment by zonzujiro** [comment]
I have installed locally last version of prettier, but looks like it's still using old version.

```PS C:\Users\Ivan\Documents\projects\phase-II> npm list prettier
cra-project@0.1.0 C:\Users\***\Documents\projects\cra-project
`-- prettier@1.7.0```

**Comment by AndrewIngram** [comment]
Likewise, my project has 1.7 installed, but this plugin is still formatting against 1.6

**Comment by wmertens** [comment]
Will be fixed by #224
